app/experimenter/nimbus-ui/src/components/PageSummary/index.tsx

```diff
@@ -3,24 +3,185 @@
     testId="PageSummary"
+    setHead={false}
     {...{ polling }}
-    analysisRequiredInSidebar
   >
-    {({ experiment, refetch }) => <Summary {...{ experiment, refetch }} />}
+    {({ experiment, refetch }) => <PageContent {...{ experiment, refetch }} />}
   </AppLayoutWithExperiment>
 );
 
+const PageContent: React.FC<{
+  experiment: getExperiment_experimentBySlug;
+  refetch: () => void;
+}> = ({ experiment, refetch }) => {
+  const [showLaunchToReview, setShowLaunchToReview] = useState(false);
+  const { invalidPages, InvalidPagesList } = useReviewCheck(experiment);
+
+  const status = getStatus(experiment);
+
+  const {
+    isLoading,
+    submitError,
+    callbacks: [
+      onLaunchToPreviewClicked,
+      onBackToDraftClicked,
+      onLaunchClicked,
+      onReviewApprovedClicked,
+      onReviewRejectedClicked,
+    ],
+  } = useChangeOperationMutation(
+    experiment,
+    refetch,
+    {
+      status: NimbusExperimentStatus.PREVIEW,
+      changelogMessage: CHANGELOG_MESSAGES.LAUNCHED_TO_PREVIEW,
+    },
+    {
+      status: NimbusExperimentStatus.DRAFT,
+      changelogMessage: CHANGELOG_MESSAGES.RETURNED_TO_DRAFT,
+    },
+    {
+      status: NimbusExperimentStatus.DRAFT,
+      statusNext: NimbusExperimentStatus.LIVE,
+      publishStatus: NimbusExperimentPublishStatus.REVIEW,
+      changelogMessage: CHANGELOG_MESSAGES.REQUESTED_REVIEW,
+    },
+    {
+      status: NimbusExperimentStatus.DRAFT,
+      statusNext: NimbusExperimentStatus.LIVE,
+      publishStatus: NimbusExperimentPublishStatus.APPROVED,
+      changelogMessage: CHANGELOG_MESSAGES.REVIEW_APPROVED,
+    },
+    {
+      status: NimbusExperimentStatus.DRAFT,
+      statusNext: null,
+      publishStatus: NimbusExperimentPublishStatus.IDLE,
+    },
+  );
+
+  const {
+    publishStatus,
+    canReview,
+    reviewRequest: reviewRequestEvent,
+    rejection: rejectionEvent,
+    timeout: timeoutEvent,
+  } = experiment;
+
+  const summaryAction = getSummaryAction(status, experiment.canReview);
+  const summaryTitle = summaryAction ? `Summary, ${summaryAction}` : "Summary";
+
+  return (
+    <>
+      <Head title={`${experiment.name} – ${summaryTitle}`} />
+
+      {submitError && (
+        <Alert data-testid="submit-error" variant="warning">
+          {submitError}
+        </Alert>
+      )}
+
+      {summaryAction && <h2 className="mt-3 mb-4 h4">{summaryAction}</h2>}
+
+      {(status.draft || status.preview) && invalidPages.length > 0 ? (
+        <Alert variant="warning">
+          Before this experiment can be reviewed or launched, all required
+          fields must be completed. Fields on the <InvalidPagesList />{" "}
+          {invalidPages.length === 1 ? "page" : "pages"} are missing details.
+        </Alert>
+      ) : (
+        <ChangeApprovalOperations
+          {...{
+            actionDescription: status.live ? "end" : "launch",
+            isLoading,
+            publishStatus,
+            canReview: !!canReview,
```

Comment:
isn't `canReview` already boolean?

Reply:
It is, this just makes doubly sure I suppose? (I did not add this, was copied from `Summary` 🤔)